charts/ratify/templates/configmap.yaml

```diff
@@ -43,6 +44,13 @@ data:
                     "name": "awsEcrBasic"
                 }
                 {{- end }}
+                {{- if $dockerAuthMode }}
```

Comment:
Currently, if not auth provider is specified, the `dockerConfig` auth provider will be selected with the same default docker config file path. Are you noticing different behavior?